### DISCUSSION TITLE
proper way to store public data

### DISCUSSION THREAD

**Discussion by ghost** [community/community#21671] [New to GitHub]
<p>Hello,</p>
<p>I am a bit confused about the proper way of storing public data. The most convenient one for me would be a directory inside the repository of the project. But is that okay with the GitHub Terms of Service?</p>
<p>Greetings,</p>
<p>P.</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>I’m not sure what you mean by “public data” in this context? As it states in our Terms of Service, you’re responsible for whatever information you store on or submit to GitHub. We don’t comment on whether some hypothetical use would or would not violate our Terms of Service.</p>
<p>I hope that helps. Let us know if you have more questions.</p>
